test.js

```diff
@@ -1,4 +1,4 @@
-var testrunner = require('qunit');
+var testrunner = require('node-qunit');
```

Comment:
so what is the big difference here? rewriting tests i was about to make the switch, but then i did not quite figure out what was the newer, and i just moved out from the old syntax of qunit 0.9.